cmd/kubeadm/app/cmd/reset.go

```diff
@@ -81,9 +112,11 @@ func (r *Reset) Run(out io.Writer) error {
 	// Don't check for errors here, since umount will return a non-zero exit code if there is no directories to umount
 	exec.Command("sh", "-c", "cat /proc/mounts | awk '{print $2}' | grep '/var/lib/kubelet' | xargs umount").Run()
 
-	dirsToRemove := []string{"/var/lib/kubelet", "/var/lib/etcd", "/etc/kubernetes"}
-	fmt.Printf("Deleting the stateful directories: %v\n", dirsToRemove)
-	for _, dir := range dirsToRemove {
+	resetConfigDir("/etc/kubernetes/")
+
+	dirsToClean := []string{"/var/lib/kubelet", "/var/lib/etcd"}
+	fmt.Printf("Deleting stateful directories: %v\n", dirsToClean)
```

Comment:
Hmm just the print? The config dir cleanup prints it's own similar message. 

Or do you mean the entire cleanup of the /var/lib dirs should occur before config dir cleanup.